x-pack/plugins/apm/e2e/cypress/integration/rum_dashboard.feature

```diff
@@ -6,18 +6,32 @@ Feature: RUM Dashboard
     Then should redirect to rum dashboard
       And should have correct client metrics
 
-  Scenario: Page load distribution
+  Scenario: Rum page filters
+    Given a user click the filter
+    When the user select the filter
+    And user applies the selected filter
+    Then it filters the client metrics
```

Comment:
Wdyt about this?

```gherkin
Scenario: Rum page filters
    When the user filters by "put-here-your-filter-name"
    Then the client metrics are filtered
```
The Given part could not be necessary, but the `When` one is key:
- Given: preparation of the test before the scenario
- When: the action that triggers the scenario
- Then: the outcome/output to be verified after the scenario

Reply:
i have updated this one as well.